packages/frontend/src/components/ProjectConnection/Inputs/MultiKeyValuePairsInput.tsx

```diff
@@ -26,8 +28,21 @@ export const MultiKeyValuePairsInput = ({
     documentationUrl,
     labelHelp,
 }: Props) => {
-    const { control } = useFormContext();
-    const { fields, remove, append } = useFieldArray({ name, control });
+    const form = useFormContext();
+
+    const values: { id: string; key: string; value: string }[] =
+        get(form.values, name) ?? [];
+
+    const addValue = () => {
+        form.insertListItem(name, {
+            id: uuidv4(),
+            key: '',
+            value: '',
```

Comment:
`id` is for mantine form to internally manage a list?

Reply:
for react keys